routing/control_tower_test.go

```diff
@@ -0,0 +1,253 @@
+package routing
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/routing/route"
+
+	"github.com/lightningnetwork/lnd/lntypes"
+)
+
+var (
+	priv, _ = btcec.NewPrivateKey(btcec.S256())
+	pub     = priv.PubKey()
+
+	testHop = &route.Hop{
+		PubKeyBytes:      route.NewVertex(pub),
+		ChannelID:        12345,
+		OutgoingTimeLock: 111,
+		AmtToForward:     555,
+	}
+
+	testRoute = route.Route{
+		TotalTimeLock: 123,
+		TotalAmount:   1234567,
+		SourcePubKey:  route.NewVertex(pub),
+		Hops: []*route.Hop{
+			testHop,
+			testHop,
+		},
+	}
+)
+
+// TestPaymentControlSubscribeUnknown tests that subscribing to an unknown
+// payment fails.
+func TestPaymentControlSubscribeUnknown(t *testing.T) {
```

Comment:
tests should be named `TestControlTower...` since we are now separating `ControlTower` and `PaymentControl` :)